plugins/async-node/components/src/dsl/async.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import type { ObjectNode } from "react-json-reconciler";
+
+export type AsyncNodeProps = {
+  id?: string;
```

Comment:
We'd probably want to have ID be mandatory if we aren't going to have a way to generate them. I would also push for them to have to be defined since content authors need to know the node id that they have in content to resolve in their plugin usage.

Reply:
done